datasets/simulated.py

```diff
@@ -11,11 +11,8 @@ class Dataset(BaseDataset):
     # List of parameters to generate the datasets. The benchmark will consider
     # the cross product for each key in the dictionary.
     parameters = {
-        'n_samples, n_features': [
-            (100, 5_000),
-            (100, 10_000)
-        ],
-        'rho': [0, 0.6],
+        "n_samples, n_features": [(100, 5_00), (100, 10_0)],
+        "rho": [0.5],
```

Comment:
same (it will break things in the doc of benchopt, it's a bad design I know.

Reply:
idem